src/pages/new-record.js

```diff
@@ -465,6 +469,10 @@ const NewRecord = () => {
               </Stack>
             </FormGroup>
 
+            <TextInput labelText="Requester name" />
+
+            <TextInput labelText="Requester phone number" />
```

Comment:
@steph237 great work. 

Here is suggestion. It is good practice to group related fields. Carbon provides a [FormGroup component](https://react.carbondesignsystem.com/?path=/story/components-formgroup--default) which is perfect for this